packages/application-shell/src/components/application-shell/application-shell.spec.js

```diff
@@ -846,7 +846,7 @@ describe('when clicking on navbar menu toggle', () => {
     });
 
     // Check that the "Support" link is rendered when the menu is expanded
-    expect(navbarRendered.getByText('Support')).toBeInTheDocument();
+    expect(navbarRendered.getAllByText('Support')).toHaveLength(1);
```

Comment:
I'm sorry but I don't understand this change. I thought we were updating the original assertion because the text were in the HTML twice, but if we're only having one, then there's no need to change the original assertion, right?

Reply:
Yes, I just left a comment on it now that I think about it. Normally, there should be nothing in the dom at this point but I am investigating why `display: none` is not showing anything, as expected, but still shows the tooltip in the dom. 
![image](https://github.com/commercetools/merchant-center-application-kit/assets/19975842/61bff7f5-347f-44ed-9a52-51ddc010b273)

Reply:
Ok, so I think I figured out what's happening: when we first render the menu expanded, the tooltip is not rendered, so there's only one "Support" text in the DOM with is the title of the menu item. When we collapse the menu, we don't render the menu item anymore, but we render the tooltip with the same text. So we always have one "Support" text.

Perhaps we can use the `tooltip` [role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role) for the tooltip so we can target the test selector using that role. _(for all menu items tooltips)_

Reply:
Yes, exactly what I am explaining here: https://github.com/commercetools/merchant-center-application-kit/pull/3432#discussion_r1515960589
>Actually here, because we are checking when expanded, we have only 1 support element in the dom from the Support label and not from the tooltip.
{!isMenuOpen && <SupportMenuTooltipContainer>...

Reply:
>Perhaps we can use the tooltip [role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role) for the tooltip so we can target the test selector using that role. (for all menu items tooltips)

@CarlosCortizasCT just to double-check what you mean here, do you mean have another test case to check for all menu item tooltips?

Reply:
What I mean is that, every time we us the Tooltip component, we should be passing the `role="tooltip"` attribute.
If we do so, then we can update the current failing test to better differentiate between the menu item label and its tooltip.

Reply:
Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3432/commits/4036a48c64968dd4531520268a00f2186803198f

Thank you